src/scss/pages/_spaces.scss

```diff
@@ -0,0 +1,206 @@
+.spaces {
+  --blue-1: #3740ff;
+  --blue-2: #3c82ff;
+  --blue-3: #3fc4ff;
+
+  audio {
+    width: 250px;
```

Comment:
There are a couple of places in this file where `px` units are used, a few others that use `em`, and then some references to sizes in variables.

That's not necessarily a problem, but I wanted to point it out in case it happened organically and you want to reconcile things.